src/components/settings/releaseNotes/ReleaseNotesDashboard.jsx

```diff
@@ -0,0 +1,110 @@
+import { Component } from 'react';
+import { observer } from 'mobx-react';
+import { Octokit } from '@octokit/core';
+import { defineMessages, injectIntl } from 'react-intl';
+import Markdown from 'markdown-to-jsx';
+import { openExternalUrl } from '../../../helpers/url-helpers';
+import { getFerdiumVersion } from '../../../helpers/update-helpers';
+
+const debug = require('../../../preload-safe-debug')(
+  'Ferdium:ReleaseNotesDashboard',
+);
+
+const messages = defineMessages({
+  headline: {
+    id: 'settings.releasenotes.headline',
+    defaultMessage: 'Release Notes',
+  },
+  connectionError: {
+    id: 'settings.releasenotes.connectionError',
+    defaultMessage:
+      'An error occured when connecting to Github, please try again later.',
+  },
+  connectionErrorPageMissing: {
+    id: 'settings.releasenotes.connectionErrorPageMissing',
+    defaultMessage:
+      'An error occured when connecting to Github, the page you are looking for is missing.',
+  },
+});
+
+class ReleaseNotesDashboard extends Component {
+  static propTypes = {};
+
+  constructor(props) {
+    super(props);
+
+    this.state = { data: '' };
+  }
+
+  async componentDidMount() {
+    const { intl } = this.props;
+
+    const octokit = new Octokit();
+    try {
+      const response = await octokit.request(
+        'GET /repos/{owner}/{repo}/releases/tags/{tag}',
+        {
+          owner: 'ferdium',
+          repo: 'ferdium-app',
+          tag: getFerdiumVersion(),
+        },
+      );
+
+      debug('GH Connection Status', response.status);
+      if (response.status === 200) {
+        const json = await response.data.body;
+        this.setState({ data: json });
+      } else {
+        this.setState({
+          data: `### ${intl.formatMessage(messages.connectionError)}`,
+        });
+      }
+
+      for (const link of document.querySelectorAll('.releasenotes__body a')) {
+        link.addEventListener('click', this.handleClick.bind(this), false);
+      }
+    } catch (error) {
+      debug('GH Connection Error Status', error);
+      this.setState({
+        data: `### ${intl.formatMessage(messages.connectionErrorPageMissing)}`,
+      });
+    }
+  }
+
+  handleClick(e) {
+    e.preventDefault();
+    openExternalUrl(e.target.href);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener(
+      'click',
+      // eslint-disable-next-line unicorn/no-invalid-remove-event-listener
+      this.handleClick.bind(this),
```

Comment:
Is this binding on unmounting?

Reply:
Yes, I followed this: https://github.com/ferdium/ferdium-app/blob/b85efe342e727905eb227bf227a9d1f06ae48e79/src/components/settings/SettingsLayout.jsx#L29-L36

Should we remove it?